superflore/generators/ebuild/overlay_instance.py

```diff
@@ -23,10 +23,12 @@
 
 
 class RosOverlay(object):
-    def __init__(self, repo_dir, do_clone, org='ros', repo='ros-overlay'):
+    def __init__(
+            self, repo_dir, do_clone, org='ros', repo='ros-overlay',
```

Comment:
nit: I think you can reduce the indent of this line to keep the `from_branch=''` on the same line -- I would prefer if you could

Reply:
Thanks! I applied the same format as regenerate_manifests below.